README.md

```diff
@@ -2,32 +2,24 @@
 Library for miscellaneous utilities used in [UDBM](https://github.com/UPPAALModelChecker/UDBM), [UCDD](https://github.com/UPPAALModelChecker/UCDD) and others.
 
 ## Dependencies
-This repository depends on `cmake gcc g++ xxHash doctest benchmark boost`.
-## Build, Test and Install
+UUtils depend on tools like `git`, `cmake`, `ninja`/`make` and a compiler installed:
 ```shell
-git clone https://github.com/UPPAALModelChecker/UUtils
-cmake -B build -S UUtils -DCMAKE_BUILD_TYPE=Release -DUUtils_WITH_TESTS=ON -DUUtils_WITH_BENCHMARK=ON
-cmake --build build
-(cd build ; ctest)
-sudo cmake --install build
+sudo apt-get -y install git cmake ninja make gcc g++
```

Comment:
the `ninja` package isn't available by default on `Ubuntu 22.04.2 LTS`, maybe it's supposed to be `ninja-build` which does exist

Reply:
Good catch, I always forget about it